bpf/lib/metrics.h

```diff
@@ -34,10 +34,10 @@ static __always_inline void update_metrics(__u32 bytes, __u8 direction,
 	entry = map_lookup_elem(&METRICS_MAP, &key);
 	if (entry) {
 		entry->count += 1;
-		entry->bytes += (__u64)bytes;
+		entry->bytes += bytes;
 	} else {
 		newEntry.count = 1;
-		newEntry.bytes = (__u64)bytes;
+		newEntry.bytes = bytes;
```

Comment:
not sure why it started complaining about this now 🤔 : https://github.com/cilium/cilium/pull/16084/checks?check_run_id=2556550015#step:4:11

Reply:
That's safe to ignore IMO. `checkpatch` is configured to only checks the changed code lines, so it only complains about this now because I touched this line.